neo4j-jdbc-bolt/src/test/java/org/neo4j/jdbc/bolt/BoltNeo4jDriverTest.java

```diff
@@ -217,13 +217,13 @@
 
 			assertEquals(PoolSettings.DEFAULT_CONNECTION_ACQUISITION_TIMEOUT, config.connectionAcquisitionTimeoutMillis());
 			assertEquals(PoolSettings.DEFAULT_IDLE_TIME_BEFORE_CONNECTION_TEST, config.idleTimeBeforeConnectionTest());
-			assertEquals((int) TimeUnit.SECONDS.toMillis( 5 ), config.connectionTimeoutMillis());
-			assertEquals(true, config.encrypted());
-			assertEquals(false, config.logLeakedSessions());
+			assertEquals((int) TimeUnit.SECONDS.toMillis( 30 ), config.connectionTimeoutMillis());
+			assertFalse(config.encrypted());
+			assertFalse(config.logLeakedSessions());
 			// assertEquals(Config.LoadBalancingStrategy.LEAST_CONNECTED, config.loadBalancingStrategy());
 			assertEquals(PoolSettings.DEFAULT_MAX_CONNECTION_LIFETIME, config.maxConnectionLifetimeMillis());
 			assertEquals(PoolSettings.DEFAULT_MAX_CONNECTION_POOL_SIZE, config.maxConnectionPoolSize());
-			assertEquals(Config.TrustStrategy.trustAllCertificates().strategy(), config.trustStrategy().strategy());
+			assertEquals(Config.TrustStrategy.trustSystemCertificates().strategy(), config.trustStrategy().strategy());
```

Comment:
Why this changed?

Reply:
This is a 4.0 change, as documented [here](https://github.com/neo4j/neo4j-java-driver/wiki/4.0-changelog#400-beta01). The change is [here](https://github.com/neo4j/neo4j-java-driver/pull/620).

Reply:
After 1 hour of looking in the driver code and the changelogs, I do not understand how the build was green before.
I started a new 4.0 build on Travis, but it's taking ages to provision a VM: https://travis-ci.org/github/neo4j-contrib/neo4j-jdbc/builds/743243669.